
public class ExceptionTest2 {

	public static void main(String[] args, char[] a) {
		
		try {
		int a=10;
		int b=0;
		
			
			System.out.println(a/b);
		}
		catch(Exception e){}
		
		System.out.println(a);

		System.out.println("예외처리된 부분 빼고 출력");
	}

}
